Documentation/ceph-fs-subvolumegroup.md

```diff
@@ -0,0 +1,44 @@
+---
+title: CephFilesystemSubVolumeGroup CRD
+weight: 3610
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](quickstart.md)
+
+# CephFilesystemSubVolumeGroup CRD
+
+Rook allows creation of Ceph Filesystem [SubVolumeGroups](https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-subvolume-groups) through the custom resource definitions (CRDs).
+Filesystem subvolume groups are an abstraction for a directory level higher than Filesystem subvolumes to effect policies (e.g., File layouts) across a set of subvolumes.
+For more information about CephFS volume, subvolumegroup and subvolume refer to the [Ceph docs](https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-volumes-and-subvolumes).
+
+## Creating daemon
+
+To get you started, here is a simple example of a CRD to create a subvolumegroup on the CephFilesystem "myfs".
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemSubVolumeGroup
+metadata:
+  name: my-subvolumegroup
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # volumeName is the name of the CephFilesystem CR
+  volumeName: myfs
+```
+
+## Settings
+
+If any setting is unspecified, a suitable default will be used automatically.
+
+### CephFilesystemSubVolumeGroup metadata
+
+- `name`: The name that will be used for the Ceph Filesystem subvolume group.
+- `namespace`: The Kubernetes namespace that will be created for the Rook cluster. The services, pods, and other resources created by the operator will be added to this namespace.
+
+
+### CephFilesystemSubVolumeGroup spec
+
+- `volumeName`: The name of the Ceph Filesystem volume.
```

Comment:
I'm not sure there is a **best** choice for what to name this. If it is `filesystem` or `filesystemName`, it doesn't match Ceph docs as closely. If it's `volumeName`, it doesn't match Rook docs as closely. 

I think it may be slightly easier for users to leave this with the Ceph terminology so that users can glean information more easily from the Ceph docs. However, doing this, I think we should be explicit here in the docs that in Rook, this is the metadata name of a CephFilesystem resource.

Reply:
I was hesitant to rename this but now that I have a better understanding (hopefully), I feel like we should rename it to `filesystemName` so this matches better with what we have in Rook and like Travis said, explaining what this means in Ceph terminology.